pkg/controller/vsphere/util.go

```diff
@@ -90,7 +92,7 @@ func getVSphereConfig(c runtimeclient.Reader) (*vSphereConfig, error) {
 	cm := &corev1.ConfigMap{}
 	cmName := runtimeclient.ObjectKey{
 		Name:      infra.Spec.CloudConfig.Name,
-		Namespace: openshiftConfigNamespace,
```

Comment:
Pass the namespace as a parameter and have something higher set it. We could have the namespace for the config as part of the actuator arguments, then it could be overridden in tests when we set the actuator up